towncrier.1

```diff
@@ -0,0 +1,168 @@
+how it was fixed, who authored the fix, and who reviewed the fix). By compiling
+a collection of these fragments, towncrier can produce a digest of the changes
+which is valuable to those who may wish to use the software.
+
+.SH COMMAND LINE OPTIONS
+
+.TP
+.B build
+Build a combined news file from news fragment. With the option
+.B \-\-draft,
+render the news fragments, don't write to files, don't check versions.
+
+.TP
+.BI check
+Check for new fragments on a branch.
+
+.TP
+.BI create " FRAGMENTFILE"
+.RS
+.P
+Create a new news fragment called \f[I]FRAGMENTFILE\f[]. Towncrier has a few standard
+types of news fragments, signified by the file extension. These are:
+.RS
+.TP
+".feature": a new feature,
+.TP
+".bugfix": a bug fix,
+.TP
+".doc": a documentation improvement,
+.TP
+".removal": a deprecation or removal of public API,
+.TP
+".misc": a ticket has been closed, but it is not of interest to users,
+.RE
+.RE
+
+.TP
+.BI \-\-config " CONFIGFILE"
+Configuration file name. (Default: towncrier will look for towncrier.toml or
+pyproject.toml file, if both files exist, the first will take precedence as
+\f[I]CONFIGFILE\f[]).
+
+.TP
+.BI \-\-date " PROJECT_DATE"
+Specify the \f[I]PROJECT_DATE\f[] following the format \f[B]YYYY-MM-DD\f[]
+(Default: current date in the system).
+
+.TP
+.BI \-\-dir " DIRECTORY"
+Create the fragment in directory \f[I]DIRECTORY\f[] (Default: \f[I]$PWD\f[]).
+
+.TP
+.B \-h
+Display usage commands (not every command line option will be printed).
+
+.TP
+.BI \-\-name " PROJECT_NAME"
+Define project name \f[I]PROJECT_NAME\f[] (Default: python package title).
+
+.TP
+.BI \-\-version " PROJECT_VERSION"
+Render the news fragments using given version " PROJECT_VERSION" (Default:
+python package version).
+
+.TP
+.B \-v
+Display towncrier version information.
+
+.SH REPORTING BUGS
+
+.P
+Please report any bugs using
+.UR https://github.com/twisted/towncrier/issues/new
+the GitHub issue tracker
+.UE .
+
+.SH SEE ALSO
+
+.P
+For more information, you may visit the
+.UR https://github.com/twisted/towncrier
+source code
+.UE .
+
+.SH AUTHOR
+\fBTowncrier\fR is copyright (c) 2015-2016, Amber Brown, meejah.
+
+.P
+This manual page was written by
+.UR sergiosacj@hotmail.com.br
+Sérgio de Almeida Cipriano Junior
+.UE .
+It was written for the Debian GNU/Linux system but may be used by others.
+
+.P
+This is free software: you may copy, modify, and/or distribute this work
+under the terms of the GNU General Public License as published by the
+Free Software Foundation; version 3 of that license or any later version.
+No warranty expressed or implied.
+
+.\" Copyright © 2021 Sérgio de Almeida Cipriano Junior <sergiosacj@hotmail.com.br>
+.
+.\" This is free software: you may copy, modify, and/or distribute this work
+.\" under the terms of the GNU General Public License as published by the
+.\" Free Software Foundation; version 3 of that license or any later version.
+.\" No warranty expressed or implied.
+.
```

Comment:
Just to get it documented and shown within the review mechanism...  Concerns have been raised about the use of the GPL here.  Discussion starts at https://github.com/twisted/towncrier/pull/326#issuecomment-796802056.